Content.Server/AlertLevel/ChangeAlertLevelComponent.cs

```diff
@@ -0,0 +1,18 @@
+using Content.Shared.ChangeAlertLevel;
+using Robust.Shared.Audio;
+
+namespace Content.Server.ChangeAlertLevel
+{
```

Comment:
Use file-scoped namespace for all new files

Reply:
do you mean like, Content.Server.AlertLevel.ChangeAlertLevel ?

Reply:
`namespace Content.Server.ChangeAlertLevel;` and get rid of the `{ }`